keylime/tpm/tpm1.py

```diff
@@ -127,9 +127,15 @@ def __run(self, cmd, expectedcode=tpm_abstract.AbstractTPM.EXIT_SUCESS, raiseOnE
         while True:
             if lock:
                 with self.tpmutilLock:
-                    retDict = cmd_exec.run(cmd=cmd, expectedcode=expectedcode, raiseOnError=False, lock=lock, outputpaths=outputpaths, env=env)
+                    retDict = cmd_exec.run(
+                        cmd=cmd, expectedcode=expectedcode,
+                        raiseOnError=False, lock=lock,
+                        outputpaths=outputpaths, env=env, shell=True)
```

Comment:
In addition to changing formatting, this is adding `shell=True` which is the opposite of what the change was trying to do right?

Reply:
Yep, there are extra works in the tpm code, using shell here is a workaround actually, because spliting them into iterables is not working from CI response at the first time, that's why I am asking if we could remove stub first, I simply want to avoid updating code that actually not in use and can be removed.